### ISSUE TITLE
Ships spawning in don't disable thrusters/don't spawn flat.

### ISSUE THREAD

**Issue by BrotherNova** [Eggs-D-Studios/wos-issues#608]
### Guidelines

- [X] I checked for duplicate bug reports
- [X] I tried to find a way to reproduce the bug

### Version

Main (Production)

### What happened? What did you expect to happen?

I flew into a planet and then when I spawned I ragdoll and fell. My ship was at an angle. The valve connecting the thruster did into shut off so when I was lucky and got someone to help, I unanchored the ship and it flew away at 1000 studs/s. If this is intended behavior I think it should be changed, valves should be shut off and ships should be spawned in as if they were stud aligned (with the seat facing up)

### How can we reproduce this bug?

Fly into space at an angle with both a vehicle seat and a spawn point on a ship that is not enclosed

### Frequency

Always

**Comment by hail12pink** [comment]
this is intended because automatically shutting off every switch in the ship on ship load would be stupid

**Comment by BrotherNova** [comment]
Yes but having a TurnOff on load button for items would be nice, especially for a valve.

**Comment by BrotherNova** [comment]
What about being oriented upright on load?

**Comment by hail12pink** [comment]
> Yes but having a TurnOff on load button for items would be nice, especially for a valve.

idk you could just like make a circuit that signals only once so it turns them off on server start. idk you could also like not turn off your anchors if you don't want your ship to move...

**Comment by hail12pink** [comment]
> What about being oriented upright on load?

you literally didn't provide any screenshots or videos of the model or bug at all so like what do you want me to say.

**Comment by hail12pink** [comment]
and the code that specifically activates anchors on load literally used to have a comment saying `-- and here, we solve runaway ships once and for all!`, like i don't know what to say because you essentially clicked the button that makes your ship fly away and then surprisingly your ship flew away

**Comment by BrotherNova** [comment]
I had no other choice cause it was kinda almost upside down in the sky. I couldn't reach anything else but the anchor from pukeko company.

**Comment by BrotherNova** [comment]
I am going to add a micro to fix my ship from spawning in a position that causes me to be softlocked.

**Comment by hail12pink** [comment]
> I had no other choice cause it was kinda almost upside down in the sky. I couldn't reach anything else but the anchor from pukeko company.

is the hyperdrive upside down or something? you literally give no information to identify what's wrong

**Comment by BrotherNova** [comment]
The ship with a upright hyperdrive spawned at the same 78° angle as I cane into the planet with.

**Comment by dogranger1** [comment]
> The ship with a upright hyperdrive spawned at the same 78° angle as I cane into the planet with.

this is basically a demo of this bug happening, where the ship keeps its orientation upon warping, whereas previously the ship would align itself to the grid upon warping.  This is an intentional addition, though it does cause some bugs, like the user not being able to spawn on any ``SpawnPoint``s properly without ragdolling - which could result in an endless loop of falling and dying. The bug seen in the video is due to the player loading before the world (whereas usually the player would load seated in their ship).

https://github.com/user-attachments/assets/6d9e7670-f4d4-49ed-aa1c-0fbbb8eba223
